zanata-war/src/main/java/org/zanata/search/LevenshteinTokenUtil.java

```diff
@@ -129,8 +129,8 @@ public static double getSimilarity(final String s1, final String s2) {
     /**
      * Splits into tokens (lower-case).
      *
-     * @param s
-     * @return
+     * @param s the string to tokenise
+     * @return an array of lowercase tokens (words)
      */
     static String[] tokenise(String s) {
         String[] tokens = s.toLowerCase().split(SPLIT_REGEX);
```

Comment:
Did you see https://bugzilla.redhat.com/show_bug.cgi?id=805737#c20 Probably fix the REGEX as well since you are here.